src/platform/user/profile/vap-svc/actions/transactions.js

```diff
@@ -203,12 +203,14 @@ export function createTransaction(
         transaction,
       });
     } catch (error) {
+      const [firstError = {}] = error.errors ?? [];
+      const { code = 'code', title = 'title', detail = 'detail' } = firstError;
```

Comment:
And in case `firstError` _is_ an empty object, this provides default values so we don't see `undefined` showing up in GA